content/applications/hr/referrals.rst

```diff
@@ -68,32 +68,34 @@ Onboarding slides can be modified if desired. Only users with :guilabel:`Adminis
 the Recruitment application can modify onboarding slides. To edit a slide, navigate to
 :menuselection:`Referrals app --> Configuration --> Onboarding.` Each line displays the text for the
 individual onboarding slide. To edit an onboarding slide, click on an individual slide line to open
-the slide's onboarding form, then click :guilabel:`Edit`.
+the slide's onboarding form.
 
-Make any changes to the message in the :guilabel:`Text` box. To change the order the slide appear in
-the onboarding, the :guilabel:`Sequence` can be modified. Type in the corresponding number to
-indicate the order in which the slides should be displayed. A :guilabel:`Company` may be selected,
-however, if this field is populated, that slide will only be displayed for that particular company.
+Make any changes to the message in the :guilabel:`Text` field. A :guilabel:`Company` may be
+selected, however, if this field is populated, that slide will only be displayed for that particular
+company.
 
-The image can be modified as well. Hover over the image thumbnail in the top right corner of the
+.. note::
+   The :guilabel:`Company` field only appears when in a multi-company database.
+
+The image can be modified as well. Hover over the image thumbnail in the top-right corner of the
 form. A :guilabel:`✏️ (pencil)` icon and :guilabel:`🗑️ (garbage can)` icon appear. Click the
 :guilabel:`✏️ (pencil)` icon to change the image. A file navigator window loads. Navigate to the
 desired image, select it, then click :guilabel:`Open`. The new image appears in the thumbnail. To
 delete an image, click the :guilabel:`🗑️ (garbage can)` icon, then select a new image using the
 :guilabel:`✏️ (pencil)` icon.
 
-To save any changes, click :guilabel:`Save`, or click :guilabel:`Discard` to delete any changes and
-revert to the original content.
-
 .. image:: referrals/edit-onboarding.png
    :align: center
    :alt: An onboarding slide in edit mode, with the main fields highlighted.
 
-The order of the slides can also be changed directly in the Onboarding dashboard, without having to
-edit an individual onboarding slide. Click the up and down arrow symbol that appears after the
-checkbox and before the slide text, and drag the slide to the desired position. The
-:guilabel:`Sequence` automatically changes on all the slide's onboarding forms, reflecting the new
-sequence.
+The slide can be saved manually at any time by clicking the *save manually* option, represented by a
+:guilabel:`(cloud upload)` icon, located at the end of the slide description in the top-left corner.
+To cancel any changes made, click the :guilabel:`↺ (Discard changes)` icon to delete any changes and
+revert to the original content.
+
+The sequence the slides appear can be changed from the Onboarding dashboard. Click the up and down
```

Comment:
```suggestion
The sequence in which the slides appear can be changed from the Onboarding dashboard. Click the up and down
```
Suggestion:
If we're on the onboarding dashboard: change to :guilabel:`Onboarding dashboard`
or, if we're not on the dashboard,
change to: *Onboarding* dashboard

Reply:
Good suggestion- we WERE but then I had people jump to the actual individual slide form. So italics itis!